### PR TITLE
Fix Bug in Claim Submit Flow

### PR DESCRIPTION
**Pull request by qwwqwwq**
- Fix api endpoint
- Fix form completion logic to consider signature as a question.

### CODE REVIEW

File: winterfell/services/claimService.js [hunk 1]
```diff
@@ -92,16 +95,16 @@ module.exports.findIncompleteClaimOrCreate = function(userId, forms, callback) {
   });
 };
 
-module.exports.addForm = function(claimId, file, callbacks) {
+module.exports.addForm = function (claimId, file, callbacks) {
```
Comment (ahsu1230) [inline winterfell/services/claimService.js]: ok I assume a space after function is standard now? haha

File: winterfell/webapp/src/js/claimSelectFormsCtrl.js [hunk 2]
```diff
@@ -9,6 +9,7 @@ app.controller('claimSelectFormsCtrl', ['$scope', 'claimService', 'formTemplateS
     });
 
     $scope.numRequiredCompleted = _.sum(_.map(forms, function (form) {
+      console.log(form);
```
Comment (ahsu1230) [inline winterfell/webapp/src/js/claimSelectFormsCtrl.js]: still need this?

**Comment by ahsu1230** [comment]
lgtm, but looks like there's a merge conflict!

File: winterfell/services/claimService.js (hunk 1 above)
Comment (qwwqwwq) [inline winterfell/services/claimService.js]: I have no idea what is standard and what is not in javascript TBH, once we turn on the linter it should all be taken care of and standardized.

File: winterfell/webapp/src/js/claimSelectFormsCtrl.js (hunk 2 above)
Comment (qwwqwwq) [inline winterfell/webapp/src/js/claimSelectFormsCtrl.js]: Nope, these statements will also be tagged by the linter for removal once we turn it on.

**Pull request merged**